app/controllers/pages_controller.rb

```diff
@@ -3,7 +3,7 @@ def home
   end
 
   def about
-    @members = Member.all
+    @members = Member.all.sort_by(&:member_role)
```

Comment:
Thinking off the top of my head here, also on my phone so excuse typos please. 

If you use an enum for the role on the member_role model, I think you could make this into two queries to enforce the order. One to get the member roles ordered by role. The other would then be getting the members and ordering them by the member_role relation that you got in the previous query. 

Might be worth a shot.